lib/realtime_web.ex

```diff
@@ -39,13 +41,15 @@ defmodule RealtimeWeb do
 
       # Include shared imports and aliases for views
       unquote(view_helpers())
+
+      unquote(verified_routes())
```

Comment:
@nbw i think this is supposed to go in `view_helpers/0` according to https://gist.github.com/chrismccord/00a6ea2a96bc57df0cce526bd20af8a7#optional-update-your-app-to-support-phoenixverifiedroutes.

Reply:
@nbw thanks! yep, i tested it on our staging and it's looking good.